applications/automatic-post-editing.md

```diff
@@ -2,6 +2,47 @@
 nav_order: 5
 parent: Application areas
 title: Automatic post-editing
-description: A method to correct errors in the machine translation output
-layout: coming_soon
+description: Automatic correction of machine translation errors
```

Comment:
I'd leave it more open, it could be used for example to add a style or locale, which aren't really errors.

e.g.

> description: Automatic post-editing of machine translation